library/Vanilla/Web/ContentSecurityPolicy/VanillaWhitelistContentSecurityPolicyProvider.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * @copyright 2009-2019 Vanilla Forums Inc.
+ * @license GPL-2.0-only
+ */
+
+namespace Vanilla\Web\ContentSecurityPolicy;
+
+/**
+ * Vanilla whitelist content security policy provider
+ */
+class VanillaWhitelistContentSecurityPolicyProvider implements ContentSecurityPolicyProviderInterface {
+
+    const VANILLA_WHITELIST = [
+        'https://www.google.com/recaptcha/api.js'
```

Comment:
This is supposed to be just the domain. AFAIK it's not possible to restrict to just a particular script. Doing this allows the whole domain anyways, so it better to be more explicit.

```suggestion
        'https://www.google.com'
```